library/include/borealis/views/button.hpp

```diff
@@ -163,6 +163,8 @@ class Button : public Box
      */
     void setState(ButtonState state);
 
+    void setText(std::string text);
```

Comment:
Can you add a documentation comment please ?

Reply:
Alright, done